relay-server/src/services/project/metrics.rs

```diff
@@ -130,7 +130,8 @@ fn is_metric_namespace_valid(state: &ProjectState, namespace: MetricNamespace) -
         MetricNamespace::Sessions => true,
         MetricNamespace::Transactions => true,
         MetricNamespace::Spans => {
-            state.has_feature(Feature::ExtractSpansAndSpanMetricsFromEvent)
+            state.has_feature(Feature::ExtractSpansFromEvent)
```

Comment:
@phacops Do we allow span metrics just because _indexed_ spans are extracted from an event?